actions.yaml

```diff
@@ -3,6 +3,16 @@ optConfig:
   optIn: true
 action: issue
 groups:
+  - name: "enable-static-check-java-repo"
+    repos:
+      - name: "opentelemetry-java-instrumentation*"
+    rules:
+      - name: "static-check-required"
+        method: "require"
+        requireAll: true
+        mustPass: false
+        actions:
+          - name: "common*"
```

Comment:
The name here needs to match a GitHub action that is used in a workflow. So for the line `uses: github/codeql-action/analyze@v3`, you could have `name: github/codeql-action/analyze`